api/src/main/java/org/openmrs/Relationship.java

```diff
@@ -9,27 +9,53 @@
  */
 package org.openmrs;
 
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
 import java.util.Date;
 
 /**
  * Relationship
  */
+@Entity
+@Table(name = "relationship")
 public class Relationship extends BaseChangeableOpenmrsData {
 	
 	public static final long serialVersionUID = 323423L;
 	
 	// Fields
-	
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "relationship_id_seq")
+	@GenericGenerator(
```

Comment:
What is the replacement for this? `unsaved-value="0"`

Reply:
this is handled by default on hibernate. isn't it?

Reply:
@dkayiwa  any update on this?